packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -601,6 +610,16 @@ export const DEFAULT_GAS_SCHEDULE: Required<FeeSchedule> = {
     [ERC20BridgeSource.Balancer]: () => 120e3,
     [ERC20BridgeSource.Cream]: () => 120e3,
     [ERC20BridgeSource.MStable]: () => 700e3,
+    [ERC20BridgeSource.MakerPsm]: (fillData?: FillData) => {
+        const psmFillData = fillData as MakerPsmFillData;
+
+        // TODO(kimpers): update with more accurate numbers after allowances have been set
+        if (psmFillData.takerToken === psmFillData.gemTokenAddresss) {
+            return psmFillData.isSellOperation ? 389e3 : 423e3;
+        } else {
+            return 444e3;
+        }
```

Comment:
What is the best way to get these numbers correct? Deploy it with high estimates, then lower once contract allowances are set?